src/Loaders/LoadingSpinner.tsx

```diff
@@ -24,11 +23,18 @@ interface Props {
    * @default "medium"
    */
   size?: Size;
+
+  /**
+   * Whether or not to sync the animation when the
+   * component is remounted, defaults to true
+   */
+  syncAnimationRestarts?: boolean;
 }
 
 export const LoadingSpinner: React.FC<Props> = ({
   theme = "light",
   size = "medium",
+  syncAnimationRestarts = true,
```

Comment:
minor: I'm sorry I keep doing this, but please add a jsdoc comment explaining what this does so we can see that from AGM. After that, merge away!